packages/react-interactive-tab/src/components/InteractiveTab/renderInteractiveTab.tsx

```diff
@@ -0,0 +1,30 @@
+/** @jsxRuntime classic */
+/** @jsx createElement */
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { createElement } from '@fluentui/react-jsx-runtime';
```

Comment:
bit confused here, why is all of this needed ? if we wanna use custom createElement factory that should be setup either within swc or if it's only per file basis via @jsx pragma no ?

Reply:
In the repository, two approaches are used: configuring the `jsx` pragma in the SWC configuration and utilizing comments. Initially, I used the latter, but have since switched to using the SWC configuration.

Reply:
yeah copy pasting "issue"

folks can you provide  guidance on custom jsx-pragma setup in contrib ? @bsunderhus @ling1726  ty

Reply:
I'd recommend using comments @dmytrokirpa! It is the approach we're currently using on all v9 components on fluent library itself, it also makes it clear for every dev that wants to dive into our codebase that we're using something different than just the regular pragma.

> bit confused here, why is all of this needed ? if we wanna use custom createElement factory that should be setup either within swc or if it's only per file basis via @jsx pragma no ? @Hotell 

In fact, we're not event supposed to support SWC only configuration, as this will fail on our storybooks (I'm not 100% sure.... it used to fail at least the last time I checked).  We don't even want to use `createElement` in this process, we want to use the most modern `automatic` pragma where you point directly to the package that has the new runtime, but again some bundler configuration is biting us right now.

Reply:
https://github.com/microsoft/fluentui/pull/32838/files

Reply:
Reverted back to use comment pragma